relion/_parser/motioncorrection.py

```diff
@@ -9,17 +9,34 @@
 )
 
 
-class MotionCorrection:
-    def __init__(self, data_dir):
-        self.relion_dir = data_dir
-        self.directory = str(self.relion_dir)
-        self.file_name = None
-        # self.job_num = ""
+class MotionCorr:
```

Comment:
Any particular reason for going for the short name?
I'm thinking the full name might be more useful as nobody would see that name (outside of debugging)

Reply:
I thought to keep it matching the relion directory names, but the longer name is possibly clearer